include/utils/utils.hpp

```diff
@@ -36,27 +36,80 @@
 #include <stdarg.h>
 #include <string.h>
 #include <string>
+#include <type_traits>
 #include <vector>
 
+namespace NumberFormatUtils
```

Comment:
I actually meant for this to be in the LittleEndian/BigEndian namespaces respectively. Could you move these there?